apps/dashboard/src/components/workflow-editor/action-picker.tsx

```diff
@@ -0,0 +1,239 @@
+import { Button } from '@/components/primitives/button';
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from '@/components/primitives/dropdown-menu';
+import { Form, FormControl, FormField, FormItem, FormLabel, FormMessage } from '@/components/primitives/form/form';
+import { Input, InputField } from '@/components/primitives/input';
+import { Popover, PopoverContent, PopoverTrigger } from '@/components/primitives/popover';
+import { Separator } from '@/components/primitives/separator';
+import { URLInput } from '@/components/primitives/url-input';
+import { buttonVariants } from '@/components/primitives/variants';
+import { cn } from '@/utils/ui';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { RedirectTargetEnum } from '@novu/shared';
+import { ComponentProps } from 'react';
+import { useForm } from 'react-hook-form';
+import { RiEdit2Line, RiExpandUpDownLine, RiForbid2Line } from 'react-icons/ri';
+import { z } from 'zod';
+
+type Action = {
+  label: string;
+  redirect: {
+    url: string;
+    type: string;
+  };
+};
+
+type Actions = {
+  primaryAction?: Action;
+  secondaryAction?: Action;
+};
+
+type ActionPickerProps = {
+  value: Actions | undefined;
+  onChange: (value: Actions) => void;
+};
+
+export const ActionPicker = (props: ActionPickerProps) => {
+  const { value, onChange } = props;
+  const primaryAction = value?.primaryAction;
+  const secondaryAction = value?.secondaryAction;
+
+  return (
+    <div className="flex items-center gap-1">
+      <div className="border-neutral-alpha-200 flex min-h-10 w-full gap-1 rounded-md border p-1 shadow-sm">
```

Comment:
please align the buttons to the right ;)

Reply:
Right! Done!